### ISSUE TITLE
Error report screen does not go away after error is sent

### ISSUE THREAD

**Issue by jidanni** [TeamNewPipe/NewPipe#1858]
If we somehow trigger an error and the error reports screen appears, when we are finished sending the error by Gmail the error report screen should go away.

**Comment by Stypox** [comment]
I disagree with this: the user may need to share it in another way (e.g. by copying to clipboard)
